src/DotNet/Writer/UniqueChunkList.cs

```diff
@@ -53,5 +53,17 @@ public UniqueChunkList()
 			chunks.Add(elem);
 			return elem.chunk;
 		}
+
+		/// <inheritdoc/>
+		public override uint CalculateAlignment() {
+			uint alignment = base.CalculateAlignment();
+			chunks.Sort(DescendingComparer.Instance);
```

Comment:
Is this a stable sort? We need to produce the same output no matter what version of .NET is used.

I quickly checked the docs and it's unstable. "This implementation performs an unstable sort" https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.sort